src/Counter/index.js

```diff
@@ -124,13 +124,13 @@ class Counter extends PureComponent {
         min={min}
         width={width}
       >
-        <Button appearance={appearance} onClick={this.handleDecrementClick}>
+        <Button appearance={appearance} onClick={this.handleDecrementClick} data-testid="decrement">
```

Comment:
Why `data-fakeid` and not `data-fake-id` ?

Reply:
@AntoninMarchardDev is part of the react-testing-library api